prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -5497,7 +5497,9 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
                 (expr.field(field), stmts)
             }
             PlaceEncoding::ArrayAccess { base, index, array_elem_ty, array_len } => {
-                panic!("ArrayAccess {{ {:?}, {:?}, {:?}, {:?} }}", base, index, array_elem_ty, array_len);
+                return Err(EncodingError::unsupported(
+                    "postprocessing array access not implemented yet"
```

Comment:
"array operations are not supported yet" is more user friendly. Some people might use the auto-deployed version of Prusti before you finish your project.